openupgrade_scripts/scripts/hr/18.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,67 @@
+---Models in module 'hr'---
+new model hr.manager.department.report [abstract]
+---Fields in module 'hr'---
+hr           / hr.department            / activity_ids (one2many)       : NEW relation: mail.activity
+hr           / hr.department            / has_read_access (boolean)     : NEW
+hr           / hr.department            / message_has_sms_error (boolean): previously in module sms
+hr           / hr.department            / website_message_ids (one2many): previously in module portal
+
+# NOTHING TO DO
+
+hr           / hr.employee              / category_ids (many2many)      : column1 is now 'employee_id' ('emp_id') [employee_category_rel]
+
+DONE: pre-migration: Rename column
+
+
+hr           / hr.employee              / distance_home_work (integer)  : NEW
+hr           / hr.employee              / distance_home_work_unit (selection): NEW required, selection_keys: ['kilometers', 'miles'], hasdefault: default
+hr           / hr.employee              / employee_type (selection)     : selection_keys is now '['contractor', 'employee', 'freelance', 'student', 'trainee', 'worker']' ('['contractor', 'employee', 'freelance', 'student', 'trainee']')
+hr           / hr.employee              / hr_icon_display (selection)   : selection_keys is now '['presence_absent', 'presence_archive', 'presence_out_of_working_hour', 'presence_present', 'presence_undetermined']' ('['presence_absent', 'presence_absent_active', 'presence_present', 'presence_to_define', 'presence_undetermined']')
+hr           / hr.employee              / hr_presence_state (selection) : selection_keys is now '['absent', 'archive', 'out_of_working_hour', 'present']' ('['absent', 'present', 'to_define']')
```

Comment:
- hr_presence_state:
'to_define' -> 'out_of_working_hour'
then, if not employee.active -> 'archive'

- hr_icon_display = 'presence_' + employee.hr_presence_state